ui/src/app/views/workspace/component.ts

```diff
@@ -32,7 +35,22 @@ export class WorkspaceComponent implements OnInit {
     this.cohortsService.getCohortsInWorkspace('123', '123').subscribe(
           cohortsReceived => {
             this.cohortList = cohortsReceived;
+            this.CohortEditService.list().then(
+              cohorts => {
+                for (const coho of cohorts) {
+                  this.cohortList.push(coho);
+                }
+              }
+            );
           });
     this.currentUrl = this.router.url;
   }
+
+  addCohort(): void {
+    this.router.navigate([this.currentUrl + '/cohort/' + this.CohortEditService.add()]);
```

Comment:
This is a but confusing -- adding is like editing a newly created but empty cohort? I might  try to structure the URLs for edit vs add separately,  like `/cohorts/123/edit` and `/cohorts/create`...